about.html

```diff
@@ -1,306 +1,295 @@
+							rel="noopener noreferrer"
+							aria-label="Connect with If These Lands Could Talk on FaceBook"
+							class="img-link">
+							<img src="/assets/icons/global/facebook.svg" alt="" />
+						</a>
+					</li>
+					<!-- Instagram -->
+					<li>
+						<a
+							href="https://www.instagram.com/iftheselandscouldtalk/"
+							target="_blank"
+							rel="noopener noreferrer"
+							aria-label="Connect with If These Lands Could Talk on Instagram"
+							class="img-link">
+							<img src="/assets/icons/global/instagram.svg" alt="Instagram" />
+						</a>
+					</li>
+				</ul>
 			</div>
 		</div>
+	</footer>
+
+	<!-- Land Acknowledgement modal -->
+	<div class="modal-overlay land-ack hidden" id="land-ack-modal">
+		<div class="modal-content">
+			<!-- button container -->
+			<div class="modal-btn-container">
+				<button
+					class="close-modal"
+					data-role="close-modal"
+					aria-label="close land acknowledgement modal">
+					<img
+						src="/assets/icons/global/close-icon-dark.svg"
+						alt="close menu icon" />
+				</button>
+			</div>
+
+			<!-- text container -->
+			<div class="modal-text-container">
+				<h4>Land Acknowledgement</h4>
+				<div class="modal-scroll-container">
+					<p>
+						ITLCT operates predominantly through digital means. The Atlanta,
+						Georgia area is the official home for ITLCT. Our leaders are
+						located in the United States between the states of Georgia and
+						Florida and so we wish to acknowledge both locations.
+					</p>
+
+					<p>
+						We acknowledge that ITLCT leaders reside on the traditional
+						homelands and territories of Florida consisting of but not limited
+						to the Seminole Nation and other first peoples such as the Calusa,
+						Tobobaga, Timucua, Apalachee, Miccosukee, Mascogo, Taíno, amongst
+						others. ITLCT also acknowledge that we operate, and our leaders
+						also reside on the traditional homelands and territories of
+						Georgia consisting of the Cherokee Nation and other first peoples
+						such as but not limited to the Catawba, Muscogee (Creek),
+						Chickasaw, Shawnee, Oconee, Apalachicola, Chatot, amongst others.
+					</p>
+
+					<p>
+						The first peoples of Florida and Georgia who lived, worked,
+						nurtured the land, and shared knowledge were involuntarily
+						removed, faced difficult challenges and oppression, may still be
+						seeking recognition and reconciliation in some cases, but today,
+						they continue to shape their communities and the future in
+						positive ways. ITLCT recognizes this and vows to be humble and
+						respectful stewards of the lands, histories, and communities of
+						Florida and Georgia's first people and other global tribes,
+						nations, and communities as we engage in our work.
+					</p>
+				</div>
+			</div>
+		</div>
+	</div>
+
+	<!-- JavaScript -->
+	<script src="/js/main.js"></script>
+	<script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>
```

Comment:
Marked helps us parse the data from the WYSIWYG editor into HTML. This is for the content sections.